docs/pages/_app.tsx

```diff
@@ -86,17 +87,28 @@ export default function MyApp({ Component, pageProps }: AppProps) {
         }
 
         @media (max-width: 800px) {
```

Comment:
Let's use breakpoint constants — it'll be easier to maintain in the future. We can copy over the JS breakpoints from Luma